Dockerfile.build

```diff
@@ -15,6 +15,7 @@ RUN yarn
 COPY intl.buildlangs.js /app/intl.buildlangs.js
 COPY static.config.js /app/static.config.js
 COPY .babelrc /app/.babelrc
+COPY .storybook /app/.storybook
```

Comment:
Can we update our serve-local process, too, to serve storybook?
And then can we delete the old Dockerfile.storybook and serve-storybook files?

Reply:
Maybe we can get storybook to serve at a different port than janis, and then we need to just let docker know to publish that port?